src/components/Product/ProductList/ProductList.tsx

```diff
@@ -78,31 +116,49 @@ export default function ProductList() {
             handleFilterLabelClick={handleFilterLabelClick}
             handleFilterByDefault={handleFilterByDefault}
           />
+        </div>
+        <div className={'mb-6 mt-1.5 flex w-full items-center justify-between'}>
+          <div
+            onClick={handleFilterClick}
+            className='hidden max-[1100px]:block cursor-pointer text-brand text-L font-medium'>
+              Filter
+          </div>
           <Dropdown<IProductSortParams>
-            className={'mb-8'}
+            className='ml-auto'
             options={sortOptions}
             onChange={handleSelect}
             selectedOption={selectedSortOption}
           />
         </div>
-        <ul
-          className={
-            'grid grid-cols-2 gap-x-2 gap-y-7 sm:gap-x-8 min-[1124px]:grid-cols-3 '
-          }
-        >
-          {data.map((product) => (
-            <li key={product.id}>
-              <ProductCard
-                id={product.id}
-                name={product.name}
-                price={product.price}
-                description={product.description}
-                productFileUrl={product.productFileUrl}
-              />
-            </li>
-          ))}
-        </ul>
-        {isShowLoadMoreBtn && (
+        <div className='w-full inline-flex gap-x-8'>
+          <FilterSidebar className='hidden min-[1100px]:block'>
+            {filtersContent}
+          </FilterSidebar>
+          { isMobileFilterOpen && <MobileFilterSidebar
+            onClose={handleCloseMobileFilter}
+            className='min-[1100px]:hidden'
+          >
+            {filtersContent}
+          </MobileFilterSidebar> }
+          <ul
+            className={
+              'grid grid-cols-2 gap-x-2 gap-y-7 sm:gap-x-6 min-[1440px]:grid-cols-3 '
+            }
+          >
+            {data.map((product) => (
+              <li key={product.id} >
+                <ProductCard
+                  id={product.id}
+                  name={product.name}
+                  price={product.price}
+                  description={product.description}
+                  productFileUrl={product.productFileUrl}
+                />
+              </li>
+            ))}
+          </ul>
+        </div>
+        {hasNextPage && !isFetchingNextPage && (
```

Comment:
Here we can replace (hasNextPage && !isFetchingNextPage) to (isShowLoadMoreBtn)
const isShowLoadMoreBtn = hasNextPage && !isFetchingNextPage (in this file)

Reply:
Fixed